_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -36,6 +40,9 @@ pipeline:
         - key: "newMessage"
           value: 3
           overwrite_if_key_exists: true
+        - metadata_key: "myMetadataKey"
```

Comment:
This value doesn't require any quotes. Let's simplify it:

```
metadata_key: myMetadataKey
```

Reply:
Changed as suggested